python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -835,11 +861,23 @@ def _generateGSObjectsFromCatalog(self, exposure, fakeCat, galCheckVal, starChec
                 gal = bulge*row['bulge_disk_flux_ratio'] + disk
                 gal = gal.withFlux(flux)
 
-                yield skyCoord, gal
+                yield skyCoord, gal, "gal"
             elif sourceType == starCheckVal:
                 star = galsim.DeltaFunction()
                 star = star.withFlux(flux)
-                yield skyCoord, star
+                yield skyCoord, star, "star"
+            elif sourceType == trailCheckVal:
+                length = row['trail_length']
+                angle = row['trail_angle']
+
+                # Make a 'thin' box profile for a trailed source
+                thickness = 1e-6
```

Comment:
How was thickness decided on? Should this be a config option? Please can you add a comment about the value.

Reply:
The thickness choice was largely arbitrary, and seemed to work empirically. I'm just looking to mimic a 'line' surface brightness profile, which isn't implemented in galsim (as far as I know). I don't think it makes sense to make configurable, but I'll add a better comment.

Reply:
@jmeyers314 once said something about this but I can't remember what it was.

Reply:
If I recall, the box is what he recommended doing.